.github/workflows/win-package-test.yml

```diff
@@ -305,10 +305,6 @@ jobs:
             throw "MSI uninstall failed"
           }
 
-      - name: "In case of failure: display the uninstall logs"
```

Comment:
Will this result in less information being available for debugging failures? (If so, is that acceptable in this case?)

Reply:
Yes, but, that will be in the case that the uninstall fails but the runner is alive :) right now the runner frequently dies in the middle of the uninstall and there are no logs anyway.